package.json

```diff
@@ -115,6 +115,7 @@
     "eslint-plugin-prettier": "~3.1.4",
     "eslint-plugin-react": "~7.21.5",
     "lerna": "^4.0.0",
+    "node-gyp": "^9.0.0",
```

Comment:
Would it be better if it was moved to `resolutions` (lines 104-107)?

Reply:
@krassowski While this does work, I avoided this because it resolves `node-gyp` to a single version:

```
node-gyp@^5.0.2, node-gyp@^7.1.0, node-gyp@^9.0.0:
  version "9.4.1"
  ...
```

Isn't this technically dangerous because [the caret specifier should only allow changes in the minor/patch version here](https://docs.npmjs.com/cli/v6/using-npm/semver#caret-ranges-123-025-004)? This might cause changes if, for whatever reason, the dependents of `node-gyp@^5` and `node-gyp@^7` don't work with `node-gyp@^9`.

I'm personally indifferent because the dev build works for me either way, as far as I can tell. I will update the PR however you suggest.

Reply:
Thanks for explanation! No strong opinion here from me I guess if it works, it works.

Reply:
Ok for this specific case, the resolution to a single version is actually better as you are sure you gonna use the same version of the build tools for all installation (and it is [independent of the NodeJS version](https://github.com/nodejs/node-gyp#node-gyp---nodejs-native-addon-build-tool)).

Moreover, as this is not a directly needed dev-dependency, I would prefer it to be a `resolution` to avoid pulling it if it is not needed.